src/app/zap-templates/zcl/data-model/chip/drlc-cluster.xml

```diff
@@ -139,70 +121,39 @@ limitations under the License.
   </struct>
 
-  <cluster apiMaturity="provisional">
+  <cluster>
     <name>Demand Response Load Control</name>
-    <domain>Energy Management</domain>
+    <domain>Energy</domain>
     <description>This cluster provides an interface to the functionality of Smart Energy Demand Response and Load Control.</description>
     <code>0x0096</code>
     <define>DEMAND_RESPONSE_LOAD_CONTROL_CLUSTER</define>
     <client tick="false" init="false">true</client>
     <server tick="false" init="false">true</server>
     <globalAttribute side="either" code="0xFFFD" value="4"/>
 
-    <features>
-      <feature bit="0" code="ENRLG" name="Enrollment Groups" summary="Supports enrollment groups">
-        <optionalConform/>
-      </feature>
-      <feature bit="1" code="TEMPO" name="TemperatureOffset" summary="Supports temperature offsets">
-        <optionalConform choice="a" more="true"/>
-      </feature>
-      <feature bit="2" code="TEMPS" name="TemperatureSetpoint" summary="Supports temperature setpoints">
-        <optionalConform choice="a" more="true"/>
-      </feature>
-      <feature bit="3" code="LOADA" name="LoadAdjustment" summary="Supports average load adjustment percentage">
-        <optionalConform choice="a" more="true"/>
-      </feature>
-      <feature bit="4" code="DUTYC" name="DutyCycle" summary="Supports duty cycle adjustment">
-        <optionalConform choice="a" more="true"/>
-      </feature>
-      <feature bit="5" code="PWRSV" name="PowerSavings" summary="Supports power savings">
-        <optionalConform choice="a" more="true"/>
-      </feature>
-      <feature bit="6" code="HEATS" name="HeatingSource" summary="Supports selecting heating source">
-        <optionalConform choice="a" more="true"/>
-      </feature>
-    </features>
-
-    <attribute side="server" code="0x0000" define="LOAD_CONTROL_PROGRAMS" type="array" entryType="LoadControlProgramStruct" writable="false">LoadControlPrograms</attribute>
+    <attribute side="server" code="0x0000" define="LOAD_CONTROL_PROGRAMS" type="ARRAY" entryType="LoadControlProgramStruct" writable="false">LoadControlPrograms</attribute>
     <attribute side="server" code="0x0001" define="NUMBER_OF_LOAD_CONTROL_PROGRAMS" type="int8u" min="5" writable="false" default="5">NumberOfLoadControlPrograms</attribute>
-    <attribute side="server" code="0x0002" define="LOAD_CONTROL_EVENTS" type="array" entryType="LoadControlEventStruct" writable="false">Events</attribute>
-    <attribute side="server" code="0x0003" define="LOAD_CONTROL_ACTIVE_EVENTS" type="array" entryType="LoadControlEventStruct" writable="false">ActiveEvents</attribute>
+    <attribute side="server" code="0x0002" define="LOAD_CONTROL_EVENTS" type="ARRAY" entryType="LoadControlEventStruct" writable="false">Events</attribute>
+    <attribute side="server" code="0x0003" define="LOAD_CONTROL_ACTIVE_EVENTS" type="ARRAY" entryType="LoadControlEventStruct" writable="false">ActiveEvents</attribute>
```

Comment:
The type of this attribute (ActiveEvents) in the XML here does not match the spec.